services/core-api/app/api/constants.py

```diff
@@ -67,6 +67,12 @@
     }
 }
 
+NOW_DOCUMENT_TEMPLATE_TYPES = {
```

Comment:
In the end we will probably want to move this to the database, along with a JSON definition of the input form. We have an interesting problem later because we will have a type, form template, and file. We could end up storing the template file in the database as well, but we'll talk about that later.

Reply:
Definitely. I figured this was just temporary.